cmd/clusterctl/examples/azure/generate-yaml.sh

```diff
@@ -28,10 +28,11 @@ RANDOM_STRING=$(date | md5sum | head -c8)
 # Azure settings.
 export LOCATION="${LOCATION:-eastus2}"
 export RESOURCE_GROUP="${RESOURCE_GROUP:-capi-${RANDOM_STRING}}"
+# TODO: Remove once existing resource group logic is handled by tagging.
+export RESOURCE_GROUP_MANAGED="${RESOURCE_GROUP_MANAGED:-yes}"
 
 # Cluster settings.
 export CLUSTER_NAME="${CLUSTER_NAME:-test1}"
-export VNET_NAME="${VNET_NAME:-}"
```

Comment:
PR says adding support for existing resource group, but i see changes for removing vnet as well, can you please revert these changes

Reply:
This variable is unused currently, hence the removal. The logic is wired up in #177 instead.